browser/src/control/Control.AutoFillOptions.ts

```diff
@@ -51,8 +51,8 @@ class AutoFillOptions extends L.Control.AutoCompletePopup {
 	}
 
 	getPopupEntries(): any[] {
-		const entries: any[] = [{ text: '', columns: [{ text: 'Copy cells' }], row: '' },
-		{ text: '', columns: [{ text: 'Fill series' }], row: '' }];
+		const entries: any[] = [{ text: 'copy', columns: [{ text: 'Copy cells' }], row: '0' },
```

Comment:
Labels should be alwasy translated:
`'abc' -> _('abc')`

Reply:
I only changed `text: 'Copy cells'` and `text: 'Fill series'` since they are only visible to user.
(`text: 'copy'` and `text: 'fill'` stay the same as before)